coremltools/converters/mil/frontend/_utils.py

```diff
@@ -158,4 +156,95 @@ def get_output_names(outputs):
         output_names = [t.name for t in outputs]
         if all([name is None for name in output_names]):
             output_names = None
-    return output_names
\ No newline at end of file
+    return output_names
+
+
+def multiply(numbers):
```

Comment:
This method is only used by `solve_generic_einsum`. So I think it should be an inner method to that method.

However I would probably just use `numpy.cumprod` instead of defining this method.